intranet/apps/auth/tests.py

```diff
@@ -27,12 +30,70 @@ def test_login_page(self):
         self.assertEqual(self.client.get(reverse("about")).status_code, 200)
         self.assertEqual(self.client.get(reverse("login")).status_code, 200)
 
-    def test_authentication(self):
-        User.objects.get_or_create(username="awilliam")
+    def add_block(self, **args):
+        block = EighthBlock(**args)
```

Comment:
Why not use `objects.create()` for these helper methods?  It saves the objects automatically.

>To create and save an object in a single step, use the create() method.

From: https://docs.djangoproject.com/en/dev/topics/db/queries/#creating-objects